applications/DEMApplication/custom_strategies/strategies/explicit_solver_continuum.h

```diff
@@ -81,6 +81,11 @@ namespace Kratos {
         void FinalizeSolutionStep() override;
         void FinalizeSolutionStepFEM();
         void MarkNewSkinParticles();
+        void ResetSkinParticles(ModelPart& r_model_part);
+        void SetSkinParticlesInnerBoundary(ModelPart& r_model_part, const double inner_radius, const double detection_radius);
```

Comment:
I propose to change this to InnerCircularBoundary (now I realized that they can be square...)

Reply:
Done!